### DISCUSSION TITLE
How much does the filer metadata store affect performance?

### DISCUSSION THREAD

**Discussion by justyns** [seaweedfs/seaweedfs#5196] [Q&A]
I'm testing out seaweedfs and am trying to figure out what impacts performance, or at least get some reasonable speeds.  I'm mostly just testing using the sysbench commands on the wiki for now.

One thing I noticed is that when using postgres (in a remote dc with 50-100ms latency), I can't get more than 5-8MiB/s.   If I switch to the leveldb3 filer store, that goes up to around 40MiB/s.

I thought having a local cache and local filer server would prevent a lot of round trips to the filer store?

How much should I expect the filer store to affect read/write performance?   Is there any reason I shouldn't stick to leveldb if it handles metadata replication/syncing already?

**Comment by NyaMisty** [comment]
I don't think your bottleneck is from filer store, try to test with a larger file and see if performance grows
